README.md

```diff
@@ -1,20 +1,62 @@
 # Certificates for CiviCRM
 
-This extension allows sites administrator to configure certificates and attached them to a CiviCRM entity (Membership, Cases, Awards, Events).
-The certificate becomes downloadable for users and site administrators when the configured condition for the certificate is satisfied.
+![Certificate](https://i.ytimg.com/vi/CRQ53B7pbss/maxresdefault.jpg)
```

Comment:
I mean I appreciate your passion 😄  , but don't think it is necessary

Reply:
though I can humbly provide an alternative design: 
![image](https://user-images.githubusercontent.com/6275540/197587376-bd70fce5-c9c2-400b-8107-439d06f578be.png)

Reply:
hahahahaha 😂, perfecto